plugins/source-kafka/README.md

```diff
@@ -94,11 +94,11 @@ kn source kafka create mykafkasrc --servers my-cluster-kafka-bootstrap.kafka.svc
 
 ### kafka delete
 
-delete NAME
+Delete a Kafka source
```

Comment:
Same here...

Reply:
change all command output to lower case, except the first word of a sentence.